src/current/_includes/v26.1/ui/statement-details.md

```diff
@@ -31,6 +31,8 @@ The **Overview** section also displays the SQL statement fingerprint statistics
 |**Execution Count** | The total number of executions. It is calculated as the sum of first attempts and retries. |
 |**Contention Time** | The amount of time spent waiting for resources. For more information about contention, see [Understanding and avoiding transaction contention]({{ link_prefix }}performance-best-practices-overview.html#understanding-and-avoiding-transaction-contention). |
 |**SQL CPU Time** | The amount of SQL CPU time spent executing the statement. The SQL CPU time represents the time spent and work done within SQL execution operators. It does not include SQL planning time or KV execution time. |
+|**KV CPU Time** | Average KV CPU time spent executing within the specified time interval. This can be thought of as [KV]({{ link_prefix }}architecture/overview.html#layers) work that is on the critical path of serving the query. It does not include any asynchronous replication related work in the [storage layer]({{ link_prefix }}architecture/storage-layer.html). |
```

Comment:
same comment as in other PR

"It does not include any asynchronous replication related work, and work in the storage layer"